transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptProcessor.java

```diff
@@ -226,6 +226,9 @@ public void operationComplete(WriteFuture future) {
 
     @Override
     protected void consume(DefaultHttpSession session) {
+        if (session.isReadSuspended()) {
+            return;
+        }
```

Comment:
Is this block required ? AbstractBridgeProcessor#updateTrafficControl0 doesn't call consume() when session  isReadSuspended.